scripts/.env

```diff
@@ -1,2 +1,4 @@
 MINIO_ACCESS_KEY=BQA2GEXO711FVBVXDWKM
 MINIO_SECRET_KEY=uvgz3LCwWM3e400cDkQIH/y1Y4xgU4iV91CwFSPC
+ROS_PROCESSOR_PORT=8004
+ROS_API_PORT=8005
```

Comment:
I think we should not add/change `ROS_API_PORT`, 
Reason -
1. If we change the `ROS_API_PORT` then we will also need to make change in the Makefile.
2. I think ros frontend by default use the port 8000 to connect to local backend while development, so that setup will so break, we will also need to make changes in frontend config. 

Better we only change `ROS_PROCESSOR_PORT` via `.env`.
```suggestion

```

Reply:
The Makefile change is already is in this pr.
I am okay to keep the 8000 as default port however I would still want ROS_API_PORT to be configurable. 
So we have the flexibility to choose another port if there is any other services already listening on it and dont require to make change in code.

Considering this I will check in frontend how to depend on the env var for ROS_API_PORT